siriuspy/siriuspy/currinfo/lifetime/main.py

```diff
@@ -122,10 +123,12 @@ def read(self, reason):
                 val_dq -= self._current_offset
 
                 # check min number of points in buffer
-                if len(val_dq) > 100:
+                if len(val_dq) > _MIN_BUFFER_SIZE:
                     fit = 'lin' if self._mode == _Const.Fit.Linear else 'exp'
                     value = self._least_squares_fit(ts_dq, val_dq, fit=fit)
                     setattr(self, lt_name, value)
+                    lt_hour = 'Lifetime' + lt_type + 'Hour-Mon'
+                    self.run_callbacks(lt_hour, value / 3600)
```

Comment:
@xresende, I didn't understand why this PV is not being updated in the same place of the code where the standard lifetime in seconds is updated.

Reply:
Hi @fernandohds564 , in fact it is, this is a loop where the Lifetime PVs are updated (with pcaspy scan)

Reply:
but where are the other `run_callbacks` calls?

Reply:
I think the original Lifetime-Mon PV is set to be scanned. Pcaspy driver class invokes this read method for it (where LifetimeHour-Mon is explicitly updated, and implicitly updates the epics DB for the invoking PV (Lifetime-Mon)

Reply:
Yes, it is correct, @xresende